README.md

```diff
@@ -116,4 +116,21 @@ clientConfig.setMaxRequests(60);
 
 profile.setHttpClientConfig(clientConfig);
 IAcsClient client = new DefaultAcsClient(profile);
+```
+
+## Timeout
+Request Settings -> Customer Settings -> Default, priority from high to low;
+The default ConnectTimeout is 5 seconds and the ReadTimeout is 10 seconds.
+```java
+//The Client timeout setting is valid for all current requests, with priority higher than the default
+HttpClientConfig clientConfig = HttpClientConfig.getDefault();
+clientConfig.setReadTimeoutMillis(readTimeoutMillis);
+clientConfig.setConnectionTimeoutMillis(connectionTimeoutMillis);
+IClientProfile profile = DefaultProfile.getProfile(regionId, accesskeyId, accesskeySecret);
+profile.setHttpClientConfig(clientConfig);
+DefaultAcsClient client = new DefaultAcsClient(profile);
+
+// Request timeout setting, valid only for current request, highest priority

```

Comment:
valid only for current request, highest priority --> valid only for the current request, has the highest priority